lib/src/js/README.md

```diff
@@ -0,0 +1,58 @@
+# JavaScript API
+
+This directory contains Dart Sass's implementation of the Sass JS API. Dart's JS
+interop support is primarily intended for _consuming_ JS libraries from Dart, so
+we have to jump through some hoops in order to effectively _produce_ a JS
+library with the desired API.
+
+JS support has its own dedicated entrypoint in [`../js.dart`]. The [`cli_pkg`
```

Comment:
Should [`../js.dart`] be  [`js.dart`] ?

Reply:
No, `js.dart` file is in the parent directory because Dart doesn't support index files.